sanic/testing.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, app):
                     session, method.lower())(url, *args, **kwargs) as response:
                 try:
                     response.text = await response.text()
+                    response.json = await response.json()
```

Comment:
So you need to have a try catch around this that will catch the `JsonDecodeError` that's causing all of the CI tests to fail.